### ISSUE TITLE
How to create and load a pre-trained MXNet model in foolbox?

### ISSUE THREAD

**Issue by GQYYY** [bethgelab/foolbox#305]
I'm going to conduct attacks on some pre-trained MXNet models with foolbox. There are no tutorials or examples in the authority docs on how to create/load an existing MXNet model. The API description and source code of **foolbox.models.MXNetModel** and **foolbox.models.MXNetGluonModel**  are vague and I still have no idea on how to use them after reading. 
Could you please give an example on how to **how to create/load an existing MXNet model in foolbox**?  To put it concretely, how to load the **.json** network file and **.params** weight file to create a MXNet model in foolbox?
Thanks!

**Comment by jonasrauber** [comment]
Instantiating your network (from a json file) or loading your weights (from a params file) sounds like an MXNet question. Once you have done that you should either have symbolic MXNet variables representing the input to the model and the output (logits) of the model or a MXNet Gluon Block instance representing the model. Those you can then pass to the `MXNetModel` class or the `MXNetGluonModel` class.
